### PR TITLE
Lesson 7 - HTML Renderer

### PR DESCRIPTION
**Pull request by phrost101**

### CODE REVIEW

File: students/phrost/Lesson07/html_render.py [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+    def __init__(self, content=None, **kwargs):
+        if content is not None:
+            raise TypeError("not allowed content")
+        super().__init__(content=content,**kwargs)
+
+
+    def render(self,out_file,cur_ind=""):
+        open_tag = ["{}<{}".format(self.indent + cur_ind,self.tag)]
+        for key, value in self.attrb.items():
+            open_tag.append(" " + key + "=" + value)
```
Comment (natasha-aleksandrova) [inline students/phrost/Lesson07/html_render.py:65]: this would be best to go into Element class

**Review by natasha-aleksandrova — APPROVED** [review]
@phrost101 great work on both assignments!

**Comment by natasha-aleksandrova** [comment]
make sure to submit on the course website to receive grades.

**Comment by phrost101** [comment]
> make sure to submit on the course website to receive grades.

Submitted thanks Natasha!!

**Pull request merged**